.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ${{ matrix.os }}
```

Comment:
Did you mean to commit this?

Reply:
https://github.com/pantsbuild/scie-pants/pull/269/commits/62cc6b23dd3d08dadfd7a767c3f1d84a53523a24

Reply:
So, yes

Reply:
I am debugging the CI failure